src/components/predictive-layer/predictive-layer.vue

```diff
@@ -1,198 +1,184 @@
 <template>
```

Comment:
There is a glitch when opening the `My History` aside on mobile.

https://user-images.githubusercontent.com/75546736/219642495-ddf3ba44-b7d1-43db-bb9d-242e62f6a6b6.mov

Reply:
This is the empathize appearing above the modal until it collapses. Giving the modal a `z-index` of around 10 fixes it (don't know if that would be the best solution)

Reply:
fixed with the solution @CachedaCodes proposed